test/Dialect/FIRRTL/merge-connections.mlir

```diff
@@ -91,13 +91,11 @@ firrtl.circuit "Test"   {
   // COMMON-LABEL:   firrtl.module private @DUT
   // COMMON-NEXT:     %p = firrtl.wire
   // COMMON-NEXT:     %0 = firrtl.subfield
-  // CHECK-NEXT:      firrtl.strictconnect %0, %x_a
+  // COMMON-NEXT:      firrtl.strictconnect %0, %x_a
```

Comment:
Nit: decrease indentation on "firrtl".

Reply:
Good catch, thanks, fixed entire block since it was wrong below and also 1-space for some reason.